packages/lesswrong/components/review/PostsListUserCommentedOn.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useState} from 'react'
+import {Components, fragmentTextForQuery, registerComponent} from '../../lib/vulcan-lib'
+import {AnalyticsContext} from '../../lib/analyticsEvents.tsx'
+import {useCurrentUser} from '../common/withUser.tsx'
+import {gql, NetworkStatus, useQuery} from '@apollo/client'
+import {FilterPostsForReview} from '@/components/bookmarks/ReadHistoryTab.tsx'
+
+const styles = (theme: ThemeType): JssStyles => ({
```

Comment:
should not have explicit `: JssStyles` return type, and `classes` prop in the component should be typed `classes: ClassesType<typeof styles>` to get type inference for the available class names